docs/usage.rst

```diff
@@ -47,16 +47,36 @@ We select the :class:`audbackend.backend.FileSystem` backend.
     audbackend.backend.FileSystem.create("./host", "repo")
 
 Once we have an existing repository,
-we can access it by instantiating the backend class.
-In addition,
-we use :class:`audbackend.interface.Unversioned`
-as an interface to the backend.
+we can access it by instantiating the backend class
+and open a connection by calling
+:meth:`audbackend.backend.open`.
+Later on,
+we will use
+:meth:`audbackend.backend.close`
+to close the connection.
+Instead of explicitly calling
+:meth:`audbackend.backend.open`
+and
+:meth:`audbackend.backend.close`
+it is good practise to use a with statement
+(not applicable in a notebook).
+
+.. jupyter-execute::
+
+    backend = audbackend.backend.FileSystem("./host", "repo")
+    backend.open()
```

Comment:
This seems a little bit strange to me. I see that there are backends that need to be opened and closed, but why should I have to open a file system backend. On those simple backends I would have expected that a simple `backend = audbackend.backend.FileSystem("./host", "repo")` should be sufficient.

Reply:
It is, except that it checks if the folder exists it does nothing. But I would still show it here since with other backends (e.g. the SQL one) it is important, so I think it is good practice to always call `open()` and `close()` (or if possible use a `with` statement).

Reply:
Having in mind that it is much more fun to work with `auglib` since you don't have to use it with `with`, I would highly appreciate if `open()`, `close()` / `with` are only used for backends that require it. For all others it should still be there (to not raise an error when using the same code as for another backend), but I would assume that a user needs nothing to implement if a backend does not need `open()`, `close()`. The later is already implemented in this pull request as we have `open()`, `close()` in `audbackend.backend.Base` that do nothing. In `audbackend.backend.FileSystem` we have implemented `_open()` at the moment, which I think we should remove. Again this then boils down to the problem at which stage and in which method we should check if a repo actually exists. E.g. if we integrate authentication into `open()` we can only do it after the connection is already established.

Reply:
Yes, implementing `_open()` and `_close()` is optional. But for some backends it can be important to call it. So I think it's good practice to always call it. I think it also makes sense to use the `open()` function to check if a repository exists or - in case of Artifactory - if a connection can be established.

Reply:
OK, I'm fine with staying with `backend.open()` here, but suggested a few changes to the text below.